geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/FeatureExporter.scala

```diff
@@ -119,8 +119,13 @@ class DelimitedExport(writer: Writer,
    case Formats.CSV => ","
    case Formats.TSV => "\t"
   }
-  lazy val dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+
   lazy val geometryFactory = JTSFactoryFinder.getGeometryFactory
+  lazy val dateFormat = {
+    val df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
```

Comment:
is this the default for all exports? If so can we make it full precision 8601 yyyy-MM-dd'T'HH:mm:ss.SSSZ or is the other format standard for geotools stuff?